package cms_8120;
import "type/cmd.proto";
option java_package = "com.hikvision.finance.fms.cms.hpp.verify";
option java_outer_classname = "RspServerLoginProto";

//服务器登陆请求响应
message RspServerLogin
{
    optional int32 cmd                      = 1;    			// CMD_RSP_SERVER_LOGIN
    optional int32 result                   = 2;    			// 验证结果,0验证通过，其它值验证失败
    optional string result_info             = 3;    			// 信息
    optional int32 authorization            = 4;    			// 软件授权类型(0-试用,1-正式)
    optional string production_version      = 5;    			// 平台版本
    optional int32 data_port                = 6;    			// 数据端口
    optional int32 ctrl_port                = 7;    			// 控制端口
    optional string config_info             = 8;    			// 服务器配置信息
    optional string trial_end_time          = 9;    			// 试用版到期时间
    optional int32 server_id                = 10;   			// 服务器id
    optional int32 is_need_sync_time        = 11;   			// 1-需要同步时间(从CMS获取时间，修改本地时间，达到本地时间与CMS时间同步)，0-不需要同步时间。
    optional int32  line                    = 12[default=0];   	// 线路，0-内网（默认线路）；1-外网；[主要用于某些模块需要输入线路号的配置]
    optional int32 language                 = 13[default=0];    // 语言类型 0-中文
	optional int32 moudle_type    		 	= 14;   			// 服务器类型
}

